org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/managers/MavenBuildSupport.java

```diff
@@ -26,23 +35,47 @@
  */
 public class MavenBuildSupport implements IBuildSupport {
 
+	private static Map<Path, String> pomDigests = new ConcurrentHashMap<>();
```

Comment:
We definitely need to serialize that map to disk, either after it changes (safer) or on workspace shutdown. Else subsequent server(JVM) starts will ignore previous hash check so it'll be a regression in behavior

Reply:
@fbricon ok, working on it. Is there a special place where eclipse.jdt.ls serializes objects to? Or maybe you can advise the path